### ISSUE TITLE
Saliency maps on other architecture 

### ISSUE THREAD

**Issue by abhigoku10** [tianzheng4/PointCloud-Saliency-Maps#3]
@tianzheng4 Hi thanks for open sourcing the code , i have few queries 

1. Can perform saliency maps on 3D object detection based architectures like sassd , complexyolo, pointnet++
2. If so what is the modification which we have to perform do u have any pointers or references which you can share 
Thanks in advance

**Comment by tianzheng4** [comment]
Yes, since performing saliency maps only need computing the gradient in the spherical coordinate system, it is applicable to any network architecture. For pointnet++, you may need modify the c++ code if you want to execute point dropping.
